plaid/sandbox/client.py

```diff
@@ -122,3 +130,20 @@ def connect_step(self, account_type, mfa, options=None):
             self.access_token = "test_{}".format(account_type)
             data['access_token'] = self.access_token
         return MockResponse(data, status_code)
+
+    def upgrade_step(self, mfa, options=None):
+        account_type = self.access_token.strip('test_')
+
+        if 'code' in institution['mfa']:
+            if mfa == '1234':
+                data = self._load_fixture('upgrade/success.json')
+                status_code = 200
+            else:
+                data = self._load_fixture('connect/invalid_mfa.json')
+                status_code = 402
+        else:
+            assert False, 'how did you get here? upgrade only produces code MFAs'
```

Comment:
this is a false assumption 
you can check out all of the occurences here
https://rollbar.com/AffirmMaster/affirm/items/13488/?item_page=2&#instances